samples/subsys/power/stm32_pm/lptim/src/main.c

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2016 Intel Corporation.
+ * Copyright (c) 2019 STMicroelectronics.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "sample.h"
+
+struct device *gpio_port, *gpio_port0, *gpio_port1, *gpio_port2;
+
+
+/* 1000 msec = 1 sec */
+#define WAIT_TIME 70
+#define SLEEP_TIME 2001
+/* measurement gives TIME VALUE +- 0.1ms */
+
+/* Application main Thread */
+void main(void)
+{
+
+	printk("\n\n*** LPTIM Management Demo on %s ***\n", CONFIG_BOARD);
+
+	gpio_port = device_get_binding(PORT);
+
+	/* Configure Button 1 as deep sleep trigger event (no Pull up on PA0) */
+	gpio_pin_configure(gpio_port, SW, GPIO_DIR_IN);
```

Comment:
Seems SW code is not used for now.

Reply:
changed sample to increase timeout when button is pressed